### PR TITLE
Respect hex support config again

### PR DESCRIPTION
**Pull request by timotheeg**
Somehow this was dropped, even though the config entries are still in the config class.

### CODE REVIEW

File: nestris_ocr/scan_strat/scan_helpers.py [hunk 1]
```diff
@@ -11,13 +11,19 @@
...
 
 PATTERNS = {
-    "score": "ADDDDD",
+    "score": "DDDDDD",
     "lines": "DDD",
-    "level": "AA",
+    "level": "TD",
     "stats": "DDD",
-    "das": "BD",
+    "das":   "BD",
```
Comment (blakegong) [inline nestris_ocr/scan_strat/scan_helpers.py]: I believe this is what `black` didn't like 😂

**Review by blakegong — APPROVED** [review]

File: nestris_ocr/scan_strat/scan_helpers.py (hunk 1 above)
Comment (timotheeg) [reply nestris_ocr/scan_strat/scan_helpers.py]: Oops 😓 . I pushed with `--no-verify` after my local precommit failed, and it was late I got the message:
```
env: python3.7: No such file or directory
```

I'll try to fix that today

Comment (timotheeg) [reply nestris_ocr/scan_strat/scan_helpers.py]: Oops 😓 . I pushed with `--no-verify` after my local precommit failed, and it was late I got the message:
```
env: python3.7: No such file or directory
```

I'll try to fix that today

Comment (timotheeg) [reply nestris_ocr/scan_strat/scan_helpers.py]: And fixed! Next PRs won't fail on lint :)

**Comment by timotheeg** [comment]
This one's ready @alex-ong . 

Naive piece stats restoration coming right after.

**Pull request merged**